8Knot/pages/index/index_callbacks.py

```
@@ -198,33 +592,69 @@ def dynamic_multiselect_options(user_in: str, selections, cached_options):
-                        port=6379,
-                        password=os.getenv("REDIS_PASSWORD", ""),
-                        decode_responses=True,
-                    )
-                    users_cache.ping()
-                    if users_cache.exists(f"{current_user.get_id()}_group_options"):
-                        user_options = json.loads(users_cache.get(f"{current_user.get_id()}_group_options"))
-                        options = options + user_options
-                        logging.info(f"Added {len(user_options)} user options from Redis")
-                except redis.exceptions.ConnectionError as e:
-                    logging.error(f"MULTISELECT: Could not connect to users-cache. Error: {str(e)}")
+            options = _get_server_options()
```

Ruff (0.14.7)

Local variable `options` is assigned to but never used

Remove assignment to unused variable `options`

(F841)